### PR TITLE
117 tms oauth provision

### LINKED ISSUE #117: OAuth provider

### PR DESCRIPTION
**Pull request by tmikeschu**
@njgheorghita @Sh1pley @riverswb @ski-climb ready for review

Adds set up for providing OAuth through doorkeeper. More work needed on testing this once we have API endpoints, but up and running now.

Additional changes:

* Add default photo to couch seed task
* Add simple user endpoint
* Add a serializer for that
* Fix couch thumbnail bug

All tests pass
Closes #117

### CODE REVIEW

**Review by riverswb — COMMENTED** [review]
Looks good, just need to fix the conflict

**Comment by tmikeschu** [comment]
@riverswb fixed!

**Review by riverswb — APPROVED** [review]

File: app/views/couches/show.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .couch-listing
   %section.info
     %article.main
-      - if @couch_default_thumb
+      - if @couch.default_thumb
```
Comment (njgheorghita) [inline app/views/couches/show.html.haml:4]: sweet!

**Comment by njgheorghita** [comment]
Great work @tmikeschu!

**Review by njgheorghita — APPROVED** [review]

**Review by riverswb — COMMENTED** [review]
Assuming Travis CI approves, I'll merge

**Pull request merged**